### PR TITLE
About us and Contact Us

### PR DESCRIPTION
**Pull request by SMCIND**

### CODE REVIEW

File: src/app/header/header.component.html [hunk 1]
```diff
@@ -19,12 +19,18 @@
...
+                    <a class="nav-link font-weight-bold" href="#about_us">ABOUT US</a>
+                </li>
                 <li class="nav-item">
                     <a class="nav-link font-weight-bold" [routerLink]="'/login'">LOGIN</a>
                 </li>
                 <li class="nav-item">
                     <a class="nav-link font-weight-bold" [routerLink]="'/register'">SIGN UP</a>
                 </li>
+                <li class="nav-item">
+                    <a class="nav-link font-weight-bold" href="#contact_us">CONTACT US</a>
```
Comment (saigoud1000) [inline src/app/header/header.component.html]: move the contact us before login. This should be order: About us, contact us, Privacy Policy(Make a new page and add the content which i have sent in the email), Login, Sign up

Comment (saigoud1000) [reply src/app/header/header.component.html]: Search is disappearing when i click on Newly added links.

**Comment by saigoud1000** [comment]
@SMCIND what did you fix? I still see the same changes.. change this order: 
ABOUT US LOGIN SIGN UP CONTACT Us ---> ABOUT US  CONTACT US LOGIN SIGN UP

**Comment by saigoud1000** [comment]
About us doesn't look good in the home page:

Create a new page for about us

**Comment by SMCIND** [comment]
fixed.. but when are u getting this?

only need to change order of tabs

File: src/app/home/home.component.html [hunk 2]
```diff
@@ -54,29 +54,39 @@ <h2 class="animated fadeInUp ">Find perfect vendor for your Celebration?</h2>
...
+<div style="height:100px;"></div>
+<div class="container vision-block pxy-5" >
+    <div class="row">
+            <h3 class="mx-auto font-weight-bold  bounceOutAnimation py-5 text-center"> ABOUT US</h3>
+        </div>
+        <div class="row">
+        <br>
+                <div class="col-sm-12  para-vision pxy-2 mx-auto">
+                <p class="text-justify pt-1">
+                        We are here to provide you a platform to connect with small businesses and service providers.
```
Comment (saigoud1000) [inline src/app/home/home.component.html]: **Change (About us text)this to** ----> We are here to provide you a platform to connect with small businesses and also service providers. We deal with all type of service that we have enclosed in our services list in our website. Our customers and business owners go through real time verification process while registering the account with us. we aim to provide a valid vendors/business partners/working professionals to our verified end customers.

**Comment by saigoud1000** [comment]
Screen Shot 2019-11-28 at 3 28 12 PM

**Comment by saigoud1000** [comment]
Change the order to ----> **About us Privacy Policy Contact Us Login Sign Up**

**Pull request merged**